examples/fingerprint_template_file_compare.py

```diff
@@ -0,0 +1,174 @@
+import serial
```

Comment:
You will need to add the License/copyright lines here -- see the other examples
I think that will satisfy the CI failure

Reply:
I was wondering why it failed. Thanks. I will add the license.

Reply:
This is done.